pkg/controller/clusterprovision/installlogmonitor_test.go

```diff
@@ -324,62 +298,16 @@ func TestParseInstallLog(t *testing.T) {
 	}
 }
 
-func buildRegexConfigMap() *corev1.ConfigMap {
-	cm := &corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      regexConfigMapName,
-			Namespace: constants.DefaultHiveNamespace,
-		},
-		Data: map[string]string{
-			"regexes": `
-- name: DNSAlreadyExists
-  searchRegexStrings:
-  - "aws_route53_record.*Error building changeset:.*Tried to create resource record set.*but it already exists"
-  installFailingReason: DNSAlreadyExists
-  installFailingMessage: DNS record already exists
-- name: PendingVerification
-  searchRegexStrings:
-  - "PendingVerification: Your request for accessing resources in this region is being validated"
-  installFailingReason: PendingVerification
-  installFailingMessage: Account pending verification for region
-- name: GCPInvalidProjectID
-  searchRegexStrings:
-  - "platform.gcp.project.* invalid project ID"
-  installFailingReason: GCPInvalidProjectID
-  installFailingMessage: Invalid GCP project ID
-- name: GCPQuotaSSDTotalGBExceeded
-  searchRegexStrings:
-  - "Quota \'SSD_TOTAL_GB\' exceeded"
-  installFailingReason: GCPQuotaSSDTotalGBExceeded
-  installFailingMessage: GCP quota SSD_TOTAL_GB exceeded
-# AWS Specific
-- name: AWSNATGatewayLimitExceeded
-  searchRegexStrings:
-  - "NatGatewayLimitExceeded"
-  installFailingReason: AWSNATGatewayLimitExceeded
-  installFailingMessage: AWS NAT gateway limit exceeded
-- name: AWSVPCLimitExceeded
-  searchRegexStrings:
-    - "VpcLimitExceeded"
-  installFailingReason: AWSVPCLimitExceeded
-  installFailingMessage: AWS VPC limit exceeded
-- name: ResourceLimitExceeded
-  searchRegexStrings:
-  - "LimitExceeded"
-  installFailingReason: ResourceLimitExceeded
-  installFailingMessage: Resource limit exceeded
-- name: InvalidCredentials
-  searchRegexStrings:
-  - "InvalidClientTokenId: The security token included in the request is invalid."
-  installFailingReason: InvalidCredentials
-  installFailingMessage: Credentials are invalid
-- name: KubeAPIWaitFailed
-  searchRegexStrings:
-  - "Failed waiting for Kubernetes API. This error usually happens when there is a problem on the bootstrap host that prevents creating a temporary control plane"
-  installFailingReason: KubeAPIWaitFailed
-  installFailingMessage: Failed waiting for Kubernetes API. This error usually happens when there is a problem on the bootstrap host that prevents creating a temporary control plane
-`,
-		},
+// buildRegexConfigMap reads the install log regexes configmap from within config/configmaps/install-log-regexes-configmap.yaml
+func buildRegexConfigMap() runtime.Object {
+	decode := serializer.NewCodecFactory(scheme.Scheme).UniversalDeserializer().Decode
+	stream, err := ioutil.ReadFile("../../../config/configmaps/install-log-regexes-configmap.yaml")
```

Comment:
Let's bindata this puppy.

Reply:
Oh, look, it [already is](https://github.com/openshift/hive/blob/c0e9a5bfbb68a790101c2f56e1fc40372f4bf102/pkg/operator/assets/bindata.go#L1559).

Reply:
To be clear, I'm asking you to use the bindata asset instead of loading up the file "by hand" here.

Reply:
bindata means we need to make update every time and it's not always clear why my changes come up. since go test runs from the source code directly, it's imo better to use the file directly.

Reply:
Our `verify` CI test already enforces that bindata is in sync.

This is what bindata is for.

Reply:
I could see the workflow of adding a new regex/test being a little confusing if you have to make bindata before running this test specifically.

Reply:
Fair enough.